docs/user-guide/persistent-volumes/index.md

```diff
@@ -20,6 +20,19 @@ A `PersistentVolume` (PV) is a piece of networked storage in the cluster that ha
 
 A `PersistentVolumeClaim` (PVC) is a request for storage by a user.  It is similar to a pod.  Pods consume node resources and PVCs consume PV resources.  Pods can request specific levels of resources (CPU and Memory).  Claims can request specific size and access modes (e.g, can be mounted once read/write or many times read-only).
 
+Administrators often need to offer and provision a variety of 
```

Comment:
These paragraphs seem out of place. Maybe a better transition from the previously ideas would help?